src/tracer.js

```diff
@@ -239,7 +241,11 @@ export default class Tracer {
         ctx.baggage = parent.baggage;
       }
 
-      ctx.traceId = randomId;
+      if (options.traceId128bit) {
+        ctx.traceId = Buffer.concat([Utils.getRandom64(), randomId]);
```

Comment:
this can probably made more efficient by allocating 16-bytes buffer, copying randomId, and filling high bytes with more random values. Saves at least one allocation.